### PR TITLE
bt-176: Add placeholder to select component

### LINKED ISSUE #176: Add placeholder to select component

### PR DESCRIPTION
**Pull request by MChornomaz**

### CODE REVIEW

**Comment by vladjsp** [comment]
I still don't see the placeholder on select. It appears when I click the select element but not in default (untouched) state.
`<Select placeholder='Options' name='email' control={control}  options={[ { value:'one', label:'one' } ]} label='label'/>`
select1
select3

**Comment by MChornomaz** [comment]
Hmm, need to check one thing. I need 20 minutes

**Comment by MChornomaz** [comment]
To fix this behavior it is necessary to pass to useAppForm default value of this select as any value but an empty string, it is an MI select problem. I suggest passing" ". Then everything works correctly

**Review by vladjsp — APPROVED** [review]

File: frontend/src/bundles/common/components/select/select.tsx [hunk 1]
```diff
@@ -17,6 +17,7 @@ type Properties<T extends FieldValues> = {
     control: Control<T, null>;
     name: FieldPath<T>;
     options: { value: string | number; label: string }[];
+    placeholder: string;
```
Comment (nikita-remeslov) [inline frontend/src/bundles/common/components/select/select.tsx]:
```suggestion
    placeholder?: string;
```

File: frontend/src/bundles/common/components/select/select.tsx [hunk 2]
```diff
@@ -35,14 +36,15 @@ const Select = <T extends FieldValues>({
...
+    placeholder,
 }: Properties<T>): JSX.Element => {
     const firstElementIndex = 0;
     const { field } = useFormController({
         name,
         control,
         defaultValue: (multiple
             ? [options[firstElementIndex].value]
-            : options[firstElementIndex].value) as PathValue<T, Path<T>>,
+            : ' ') as PathValue<T, Path<T>>,
```
Comment (nikita-remeslov) [inline frontend/src/bundles/common/components/select/select.tsx]: why do you need this change?

File: frontend/src/bundles/common/components/select/select.tsx [hunk 3]
```diff
@@ -35,14 +36,15 @@ const Select = <T extends FieldValues>({
     multiple,
     hasError,
     isDisabled,
+    placeholder,
```
Comment (nikita-remeslov) [inline frontend/src/bundles/common/components/select/select.tsx]:
```suggestion
    placeholder = "Placeholder",
```

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/common/components/select/select.tsx (hunk 2 above)
Comment (MChornomaz) [reply frontend/src/bundles/common/components/select/select.tsx]: They were not really necessary, fixed

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**